docs/how-to/monitor-your-backups.md

```diff
@@ -562,6 +563,58 @@ Heartbeat Retry = 360          # = 10 minutes
 Resend Notification every X times = 1
 ```
 
+## zabbix hook
+
+<span class="minilink minilink-addedin">New in version 1.8.15</span>
```

Comment:
This may post-date your initial work on this PR, but `1.8.15` is no more and will be replaced with `1.9.0`!

Reply:
https://github.com/borgmatic-collective/borgmatic/pull/85/commits/78f81c7b738affee762a8aafb7df38d2f526bf79